_posts/en/newsletters/2018-09-11-newsletter.md

```diff
@@ -103,12 +103,12 @@ wait until version 0.18 in about six months from now.*
   automatically-selected peers.  Several merges also updated which
```

Comment:
Maybe this LND merge should also be updated?

Reply:
I'm adding bolding so the bullet is hyperlinked, but I don't think we need to add links to all the merges.  (I was wondering if there was a typo s/merges/commits/ but, no, there actually were 29 merges in that period: `cd lnd ; git log --since=2018-09-01 --until=2018-09-11 --merges --oneline  | wc -l`)

Reply:
Sounds good.